xml/System/ObsoleteAttribute.xml

```diff
@@ -247,16 +249,18 @@
           <see langword="true" /> if the obsolete element usage generates a compiler error; <see langword="false" /> if it generates a compiler warning.</param>
         <summary>Initializes a new instance of the <see cref="T:System.ObsoleteAttribute" /> class with a workaround message and a Boolean value indicating whether the obsolete element usage is considered an error.</summary>
         <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- The following table shows the initial property values for an instance of <xref:System.ObsoleteAttribute>.  
-  
-|Property|Value|  
-|--------------|-----------|  
-|<xref:System.ObsoleteAttribute.IsError%2A>|The `error` value.|  
-|<xref:System.ObsoleteAttribute.Message%2A>|The `message` value.|  
-  
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The following table shows the initial property values for an instance of <xref:System.ObsoleteAttribute>.
+
+|Property|Value|
+|--------------|-----------|
+|<xref:System.ObsoleteAttribute.IsError%2A>|The `error` value.|
+|<xref:System.ObsoleteAttribute.Message%2A>|The `message` value.|
+|<xref:System.ObsoleteAttribute.DiagnosticId|`null`.|
+|<xref:System.ObsoleteAttribute.UrlFormat|`null`.|
+
  ]]></format>
         </remarks>
```

Comment:
_This comment really applies down on line 292, but I can't add the comment there._

The `DiagnosticId` property's docs still have:

```xml
<remarks>To be added.</remarks>
```

I propose we update that to:

```xml
<remarks>The diagnostic ID is shown in build output for warnings and errors, and this is the unique ID that would be used to suppress the warnings or errors if needed.</remarks>
```